homeassistant/components/dsmr/const.py

```diff
@@ -271,7 +271,7 @@
         state_class=SensorStateClass.TOTAL_INCREASING,
     ),
     DSMRSensorEntityDescription(
-        key=obis_references.BELGIUM_HOURLY_GAS_METER_READING,
```

Comment:
The name of this constant has changed, however, its contents has not. Therefore this is not a breaking change (as the unique ID remains the same).

Reply:
That's an interesting fix. https://github.com/ndokter/dsmr_parser/pull/105